index.html

```diff
@@ -1,14 +1,100 @@
+                      <a href="continue-reading.html"><button id="reading">Continue Reading</button>  </section></a>
+        </div>
+<!--My photos wouldn't upload to VS Code, so I used imgur to upload all my own photos and use those urls to share my photos here-->
+<!--Each of these blog post links goes to the same post-->
+        <div id="sidebar">
+            <a href="blog-post.html"><button id="date">July 30</button></a>
+            <h4>Just Another Post</h4>
+            <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis<hr>
+            <a href="blog-post.html"><button id="date">July 28</button></a>
+            <h4>And Another Post</h4>
+            <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis<hr>
+            <a href="blog-post.html"><button id="date">July 24</button></a>
+            <h4>One More Post</h4>
+            <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus<hr>
+            <h4>Something of Note</h4>
+            <img src="https://i.imgur.com/uxJLfFs.png" alt="Mount Crested Butte" class="side";/>
+             Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+            <a href="learn-more.html"><br><button id="more">Learn More</button></a><hr>
+            <h4>Something of Less Note</h4>
+            <img src="https://i.imgur.com/SMwTgQ5.png" alt="Black Canyon of the Gunnison" class="side";/>
+            Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+            <a href="learn-more.html"><br><button id="more">Learn More</button></a>  
+        </div>
+
+   </div>
+   <!--I made the footer and reused it on each page, creating a grid within a grid for each instance. -->
+<div id="footer">
+    <section id="questions">Questions or comments? Get in Touch:</section>
+    <section id="form"> 
+      <input type="text" placeholder="Name" id="input1"></input>
+      <input type="text" placeholder="Email" id="input2"></input>
+      <textarea type="text" placeholder="Message">Message</textarea>

```

Comment:
Putting the text "Message" between your two `<textarea>` tags puts the text onto your page rather than just using your placeholder text. If you remove this it will match the styling of your other inputs.